ansible/roles/jenkins-worker/templates/start.j2

```diff
@@ -4,8 +4,8 @@ export NODE_TEST_DIR="$HOME/tmp"
 export JOBS="{{ jobs_env }}"
 
 export OSTYPE=osx
-export ARCH=x64
-export DESTCPU=x64
+export ARCH="{{ archtype[os] }}"
+export DESTCPU="{{ archtype[os] }}"
```

Comment:
Does 
```suggestion
export ARCH="{{ arch }}"
export DESTCPU="{{ arch }}"
```
not work as expected?

Reply:
You should be able to run `ansible/plugins/inventory/nodejs_yaml.py` to see what `arch` gets set to for the M1 hosts.

Reply:
I knew there was an option but I didnt look very hard tbh will have to try that

Reply:
`nodejs_yaml.py` does spit out `arm64` so Ive switch the code to use `arch` instead